accepted/00076-configuration-channels-visibility.md

```diff
@@ -0,0 +1,201 @@
+
+Adding complexity to the scenario described above:
+
+1. create another System Group G2
+2. create another System Group G3
+3. create a Configuration State Channel C4 at System group G2
+4. create a Configuration State Channel C5 at System Group G2
+5. create a Configuration State Channel C6 at System Group G3
+6. check the following system groups channels assignments:
+  - G1 --> C2
+  - G2 --> C4
+  - G2 --> C5
+  - G3 --> C6
+7. add the System S1 to System Groups G2 and G3
+8. go to System Detail > States > Configuration Channels > System
+9. still only the C3 channel is listed as an assignment to S1, but in reality S1 is inheriting C2 from G1, C4 and C5 from G2 and C6 from G3
+
+
+## What is the expected outcome?
+
+A transparent representation of all Configuration State Channels assigned to a system from all the possible sources: direct and inherited (from System Groups, from Organization and from the system registration).
+
+## Limitation
+
+It turns out that once the Salt States are collected from the various source Configuration States Channels, they are handled by the Salt engine itself who builds up the Salt highstate, and this one has no tracking information of the order or the source of the states origin.
+
+The proposed solution of this RFC is not including an end-to-end description of where each single Salt State comes from where at the moment, the initial improvement is to add the visibility of the set of Configuration State Channels every system inherits from other entities (System Groups and Organizations as described before).
+
+
+# Detailed design
+[design]: #detailed-design
+
+### `Home > My Organization > Configuration Channels`
+This page represents the assignment connection between an organization and configuration channels. All the systems in this organization will inherit the selection set of channels here. This should be only a list of Configuration Channels with the possibility of simply select/deselect channels and Apply changes. If there is a need to show only the channel selected, a filter to the list page should be added instead.
+
+Before:
+![My Organization BEFORE](images/configuration-channels-visibility/Home_MyOrganization_ConfigurationChannels_BEFORE.png)
+
+After:
+![My Organization AFTER](images/configuration-channels-visibility/Home_MyOrganization_ConfigurationChannels_AFTER.png)
+
+### `Systems > Details > States > Configuration Channels`
+Before:
+![System Details BEFORE](images/configuration-channels-visibility/Systems_Details_States_ConfigurationChannels_BEFORE.png)
+
+After:
+![System Details AFTER 1](images/configuration-channels-visibility/Systems_Details_States_ConfigurationChannels_AFTER_1.png)
+![System Details AFTER 2](images/configuration-channels-visibility/Systems_Details_States_ConfigurationChannels_AFTER_2.png)
```

Comment:
You're right, I think we can do the same here. In the end the _three tabs_ pattern is useful when it comes to packages (`Systes > Details > States > Packages`) because in there it is nice to see changes and what is currently on the system. But in this case we just need one list where the user can check/uncheck channels, confirm, reorder and save. I'll update the screenshot accordingly. :+1:

Reply:
And same as before, the one tab left will be no-tab in the end.